src/content/development/building-testing/ci-maintenance/_index.en.md

```diff
@@ -65,4 +65,16 @@ For example, in the website repository,
           fetch-depth: 0
 ```
 
-should be checked by looking at the `checkout` [releases](https://github.com/actions/checkout/releases).
+Would be updated by this Dependabot config:
+
+```yaml
+---
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: daily
+```
+
+Hash-based versions should always be used to ensure there are no changes without an update on our side.
```

Comment:
It would be useful to mention that, when problems occur with GHAs, people should look for moved tags, as happened with https://github.com/gaurav-nelson/github-action-markdown-link-check, and consider bumping to the corresponding sha even when there isn’t an explicit non-moving tag.